Sources/Momento/internal/PubsubClient.swift

```diff
@@ -15,57 +15,35 @@ protocol PubsubClientProtocol {
 
     func subscribe(
         cacheName: String,
-        topicName: String
+        topicName: String,
+        resumeAtTopicSequenceNumber: UInt64?
     ) async throws -> TopicSubscribeResponse
     
     func close()
 }
 
 @available(macOS 10.15, iOS 13, *)
 class PubsubClient: PubsubClientProtocol {
-    var logger: Logger
-    var configuration: TopicClientConfigurationProtocol
-    var credentialProvider: CredentialProviderProtocol
-    var sharedChannel: GRPCChannel
-    var eventLoopGroup = PlatformSupport.makeEventLoopGroup(loopCount: 1)
-    var client: CacheClient_Pubsub_PubsubAsyncClient
+    let logger = Logger(label: "PubsubClient")
+    let configuration: TopicClientConfigurationProtocol
+    let credentialProvider: CredentialProviderProtocol
+    let eventLoopGroup = PlatformSupport.makeEventLoopGroup(loopCount: 1)
+    let headers = ["agent": "swift:0.1.0"]
```

Comment:
Just noticed we haven't updated this.
```suggestion
    let headers = ["agent": "swift:0.3.2"]
```